ansibullbot/utils/moduletools.py

```diff
@@ -862,6 +862,8 @@ def get_module_authors(self, module_file):
     def extract_github_id(self, author):
         authors = set()
 
+        if author is None:
+            return list()
```

Comment:
I have always wondered about the exact advantage/disadvantage to using literals vs `list()`. Can you educate me?

Reply:
First, it's more pythonic and readable in general. But I bet it's also faster because `[]` is not a function call. Probably @abadger has smth to add.

P.S. You can just "accept" code suggestions in PRs now and GitHub will put those into your branch. Non need for manual commit :)